pt-BR/mail/chrome/messenger/am-server-top.dtd

```diff
@@ -2,12 +2,12 @@
 
 <!ENTITY serverSettings.label "Receber mensagens">
 <!-- LOCALIZATION NOTE (popServer.label) : do not translate "POP" in below line -->
-<!ENTITY popServer.label "POP Mail Server">
+<!ENTITY popServer.label "Servidor de emails POP">
 <!-- LOCALIZATION NOTE (imapServer.label) : do not translate "IMAP" in below line -->
-<!ENTITY imapServer.label "IMAP Mail Server">
+<!ENTITY imapServer.label "Servidor de emails IMAP">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY imapServer.label "Servidor de mensagens IMAP">